README.md

```diff
@@ -2,20 +2,21 @@
 
 You have a process which is done manually or even on paper and you want to move into the digital world.
 
-Your process has the following characteristics:
+Your process has...
 
-* Centered around a form (e.g. application, certification, ...)
-* Different users involved
-* Specific flow how a form is processed
+* a form based focus (e.g. application, certification, ...)
+* different users and roles involved
+* a specific workflow of processing and decision-making
```

Comment:
with a workflow most people envision a fix flow how to get from a to z.
We wanted to avoid this because what is possible in Caluma is much more than a workflow.. More of a task flow. Have to think how we could put this into text.

Reply:
I agree! Task flow would be a better wording as we want to avoid to force Caluma into specific fields where a "workflow" is applied.